src/frontend/js/media-management/library.tsx

```diff
@@ -237,17 +247,26 @@ export default function Library({
           onClick={() => setFileIndex(null)}
         >
           <div class="rounded w-full bg-blue-500 text-white font-bold">
-            <Breadcrumbs breadCrumbs={directoryPath} mediaTranslations={mediaTranslations} />
-          </div>
-          <div class="p-4">
-            <DirectoryContent
-              fileIndexState={[fileIndex, setFileIndex]}
-              directoryContent={directoryContent}
+            <Breadcrumbs
+              breadCrumbs={directoryPath}
               mediaTranslations={mediaTranslations}
-              selectionMode={selectionMode}
-              globalEdit={globalEdit}
             />
           </div>
+          {isLoading ? (
+            <div class="flex h-screen">
+            <Loader class="animate-spin m-auto w-1/6 h-1/6"></Loader>
+            </div>
```

Comment:
```suggestion
            <Loader class="absolute w-32 h-32 -mt-9 -ml-16 inset-1/2 text-gray-600 animate-spin"/>
```
I think it looks a bit strange when the loading screen exceeds the available screen space and one needs to scroll into empty space... Maybe this looks a bit better?
This just requires the `relative` class on the [parent element](https://github.com/Integreat/integreat-cms/blob/b2f3fce6d97fe28946b38559d12f6f7afdc59323/src/frontend/js/media-management/library.tsx#L246).